docbook/reference/en/en-US/modules/MicroProfile_Rest_Client.xml

```diff
@@ -15,6 +15,21 @@ In particular, the API code is in
 and the specification is in
 <ulink url="https://github.com/eclipse/microprofile-rest-client/tree/master/spec">https://github.com/eclipse/microprofile-rest-client/tree/master/spec</ulink>.
 </para>
+<para>
+    <emphasis role="bold">IMPORTANT:</emphasis> As of RESTEasy 5.0.0 the MicroProfile integration has moved to a new
+    project, group id, artifact id and version. The new group id is <code>org.jboss.resteasy.microprofile</code>.The
+    current released version is 1.0.0.Alpha1. The artifact id for the client is now <code>microprofile-client</code>.
```

Comment:
Just a minor note to say that mentioning versions this way in the doc is risky, as next time we release chances are that we forget to update this.